### ISSUE TITLE
Starting work on Documentation. 

### ISSUE THREAD

**Issue by Prakash4844** [Prakash4844/tit#18]
@sagargoswami2001 @SakuraBlossomTree how should we approach the Documentation part?

**Comment by SakuraBlossomTree** [comment]
Iam thinking we create a documentation like how git has

**Comment by Prakash4844** [comment]
I think git-scm site uses ruby on rails with SQLite. It actually a web app. 

It would be overkill for this site. And having a Database would mean we won't be able to host it for free on static site hosting services like netlify, gh pages or vercel.

I think it would be better to have a minimal static site.

**Comment by SakuraBlossomTree** [comment]
yes something like that

**Comment by SakuraBlossomTree** [comment]
beacuae it will be much more better than actually keeping it in the README.md

**Comment by Prakash4844** [comment]
I am thinking we could use something like
- https://gohugo.io
- https://docusaurus.io/
- https://docsify.js.org/#/
- https://www.mkdocs.org/

**Comment by Prakash4844** [comment]
Do you know any of the above option? Or something similar?

**Comment by Prakash4844** [comment]
Yes, i think basic Hugo site would be enough for now. should we start a separate repo for site?

**Comment by SakuraBlossomTree** [comment]
Sorry, was busy with things

But yes we can keep a separate repo for that

**Comment by Prakash4844** [comment]
No worries, actually i am graduating very soon, so i am focusing on getting a job first. so the development on this might be real slow.

**Comment by SakuraBlossomTree** [comment]
Thats good for you 

All the best for your near future 

I will contribute for this project

**Comment by Prakash4844** [comment]
Thank-you.

**Comment by SakuraBlossomTree** [comment]
Hey are you still working on this project?

**Comment by Prakash4844** [comment]
Hey @SakuraBlossomTree, i have got a job, so I am not getting the time to be active on GitHub. Since it is an SRE role and I was fresher. I need to learn a lot for the job. I plan on working on this eventually but currently this is on Halt.

Although all the code i have written can be seen on tit_add branch.
